deepometry/parse.py

```diff
@@ -33,7 +37,7 @@ def parse(pathname, output_directory, size, channels=None):
     if ext == ".cif":
         return _parse_cif(pathname, output_directory, size, channels)
 
-    raise NotImplementedError("Unsupported file format: {}".format(ext))
+    raise NotImplementedError("Expected file format: {}".format(".CIF"))
```

Comment:
I envisioned `parse` being a top-level function which delegated to different parse functions based on the file extensions. E.g.,

```python
def parse(pathname, output_directory, size, channels=None):
    ext = os.path.splitext(pathname)[-1].lower()

    if ext == ".cif":
        return _parse_cif(pathname, output_directory, size, channels)

    if ext == ".npy":
        return _parse_npy(pathname, output_directory, size, channels) 

    raise NotImplementedError("Unsupported file format: {}".format(ext))
```

I see how it would be annoying to call parse for every .TIF image. Maybe the first argument (`pathname`) could take either a single file name or a list of file names.